test/chrome_test.js

```diff
@@ -258,4 +292,114 @@ describe("StorageArea", function() {
       });
     });
   });
+
+  describe("sync process", function() {
+    let changeSpy;
+
+    beforeEach(function() {
+      //TODO: make this nicer once https://github.com/Kinto/kinto-chrome/issues/20
+      //is fixed:
+      if (changeSpy) {
+        changeSpy.reset();
+      } else {
+        changeSpy = sinon.spy();
+        storage.onChanged.addListener(changeSpy);
```

Comment:
We should unit test StorageArea constructor and create a fresh one before each test instead.